include/materials/material_utility.tcc

```diff
@@ -0,0 +1,230 @@
+//! Compute mean stress p
+inline const double mpm::material_utility::compute_mean_p(
+    const Eigen::Matrix<double, 6, 1>& stress) {
+  // Compute mean p in compresion positive
+  return (-1. / 3. * (stress(0) + stress(1) + stress(2)));
+}
+
+//! Compute deviatoric stress
+inline const Eigen::Matrix<double, 6, 1>
+    mpm::material_utility::compute_deviatoric_stress(
+        const Eigen::Matrix<double, 6, 1>& stress) {
+
+  // Compute mean p in tension positive
```

Comment:
Could we please have compression positive?

Reply:
OK, it has been changed